### ISSUE TITLE
I got more false positives than default model with all classes

### ISSUE THREAD

**Issue by masterchop** [zlingkang/mobilenet_ssd_pedestrian_detection#1]
I tried your model on my code to see if my pi can go faster as its taking like 1.3 frames per second but i got same performance but less tracking any reasons why?

**Comment by zlingkang** [comment]
@masterchop 

- False positives problem: 
I trained it with Caltech pedestrian dataset and only used the 'person' category label, while the dataset actually contains four categories (person, person-fa, person?, people), mark those regions as non-used region before training might be helpful. And also my GPU's memory is limited (2GB), the batch-size was 2 during the training, higher would be helpful. Given the batch-size of 2, I only did 76000 iterations which is probably not good enough.

- Speed problem: 
Instead of using Caffe on your pi, convert the models to ncnn (https://github.com/Tencent/ncnn) will possibly give you around 3x faster. And I have already done that, and I have combined the whole thing with a tracker to make it even faster in this repo(https://github.com/zlingkang/pedestrian_detection_and_tracking). If you don't need the tracking part, just run ./detection_test in the build/pedestrian_detection folder after you have compiled the whole thing.
